signal_test.go

```diff
@@ -0,0 +1,490 @@
+package dbus_test
```

Comment:
Folders are intended to map to a single package, so we need to figure out how to make this work. If the tests for the `dbus` package require something in the `introspect` package, that then depends back on the `dbus` package, there are three paths forward:
1. move the tests into the `introspect` package
2. combine the `introspect` and `dbus` packages
3. break the dependency in the `introspect` package on `dbus`

I'd prefer option 2